modules/compiler/utils/source/pass_functions.cpp

```diff
@@ -73,20 +73,36 @@ uint64_t computeApproximatePrivateMemoryUsage(const llvm::Function &fn) {
   return bytes;
 }
 
-void remapConstantExpr(llvm::ConstantExpr *expr, llvm::Constant *from,
-                       llvm::Constant *to) {
+template <typename ConstantT>
+static llvm::SmallVector<llvm::Constant *, 8> getNewOps(ConstantT *constant,
```

Comment:
Perhaps this would be better as returning `llvm::SmallVectorImpl<llvm::Constant *>` to avoid the API and consumers assuming a specific size?